configure-ironic.sh

```diff
@@ -13,6 +13,15 @@ IRONIC_FAST_TRACK=${IRONIC_FAST_TRACK:-true}
 # Whether cleaning disks before and after deployment
 IRONIC_AUTOMATED_CLEAN=${IRONIC_AUTOMATED_CLEAN:-true}
 
+# Timeout (seconds) to wait for a callback from a deploy ramdisk
+IRONIC_DEPLOY_CALLBACK_TIMEOUT=${IRONIC_DEPLOY_CALLBACK_TIMEOUT:-"4800"}
```

Comment:
Considering the comment you pointed out, we might also want to put the pxe deploy timeout customized here with the same note about its dependency

Reply:
@maelk this is the comment I was referring to in the [ironic.conf file](https://github.com/metal3-io/ironic-image/blob/e1d39417015f5a7799a42a91a36f7005d872b562/ironic.conf#L55) and I have made it customizable as you suggested.